### PR TITLE
Update gulpfile.js

### PR DESCRIPTION
**Pull request by tedgeving**
Added error logging to SASS task.   
Default Gulp task will continue to run instead of exiting on error.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -30,12 +30,14 @@ gulp.task('browserSync', function() {
...
 gulp.task('sass', function() {
-  return gulp.src('app/scss/**/*.scss') // Gets all files ending with .scss in app/scss and children dirs
-    .pipe(sass()) // Passes it through a gulp-sass
-    .pipe(gulp.dest('app/css')) // Outputs it in the css folder
-    .pipe(browserSync.reload({ // Reloading with Browser Sync
-      stream: true
-    }));
+    return gulp.src('app/scss/**/*.scss')  // Gets all files ending with .scss in app/scss and children dirs
+        .pipe(sass({
+            style: 'compressed',
```
Comment (zellwk) [inline gulpfile.js]: Please remove style. This is a preference and shouldn't be imposed on others, especially beginners.

Comment (zellwk) [reply gulpfile.js]: Also, please make sure indentation remains the same. Thanks!

**Comment by zellwk** [comment]
@tedgeving just wondering, why did you close this?

**Comment by tedgeving** [comment]
Ended up using my work account to revise the pull request, ugh. Need to keep that separate from OSS contributions. 

Any chance you can close the pull request or delete the one from k2md? I can make a new pull request for the issue if you like. 

Sorry for the confusion about this. 

Thanks
Ted

**Comment by zellwk** [comment]
This is already closed, so please feel free to create a new one

**Pull request closed without merging**